src/js/control-bar/progress-control/seek-bar.js

```diff
@@ -78,7 +78,9 @@ class SeekBar extends Slider {
     if (document.hidden) {
       this.disableInterval_(e);
     } else {
-      this.enableInterval_();
+      if (!this.player_.paused() && !this.player_.hasClass('vjs-waiting')) {
```

Comment:
I'm not sure that we should end up paused with vjs-waiting.

Reply:
Also, seems like this is unrelated to the actual issue, just a nice improvement?

Reply:
Yes, it's a improvement.It's no need to call update if the player is paused or waiting.